cg/services/orders/order_summary_service/utils.py

```diff
@@ -37,6 +38,7 @@ def create_order_summary(
         cancelled=analysis_summary.cancelled,
         failed=analysis_summary.failed,
         completed=analysis_summary.completed,
+        failed_sequencing_qc=case_summary.failed_sequencing_qc,
```

Comment:
Cigrid needs to be updated as to not always render this status (when the count is None/0)